src/app/js/formats/uri/component.js

```diff
@@ -0,0 +1,37 @@
+import React, { PropTypes } from 'react';
+import { Link } from 'react-router';
+import { field as fieldPropTypes } from '../../propTypes';
+
+const UriComponent = ({ linkedResource, resource, field }) => {
```

Comment:
i'll call it UriView, and don't forget the uppercase on the file name, all component start with one.